opencv_dnn/cpp/detect-image.cpp

```diff
@@ -74,6 +75,9 @@ int main(int argc, char* argv[]) {
 
     // Draw and display
     draw(img, nms_dets);
+    if (argc==4 && std::string(argv[3]) == "vis") {
+        vis = true;
+    }
```

Comment:
Place this block under `bool vis = false`.
Condition: `argc==4 && std::string(argv[3]) == "vis"` -> `argc==4 && std::string(argv[3]) == "true"`.